lib/web_minion/bots/capybara_bot.rb

```diff
@@ -133,15 +133,19 @@ def save_value(target, value, _element, val_hash)
     def get_form(target, _value, _element)
       if target.is_a?(Hash)
         type, target = target.first
-        return @bot.find(type, target)
-      elsif target.is_a?(String)
-        index = %w(first last).index(target)
-        return @bot.find(target) if index < 0
+        case type.to_sym
+        when :class
+          target = ".#{target}"
+        when :id
+          target = "##{target}"
```

Comment:
No, it should raise an error. Depending on the error, we'll need to cover more cases here.